keras_cv/metrics/coco/recall_test.py

```diff
@@ -21,7 +21,7 @@
 
 
 class COCORecallTest(tf.test.TestCase):
-    def test_runs_inside_model(self):
+    def DISABLE_test_runs_inside_model(self):
```

Comment:
Do we have to disable those tests?

Reply:
COCO metrics are entirely disabled in this PR as there is:

- no way to make them work with the current implementation
- they still have numerical issues.

I figured it is better to keep this PR as small as possible and just disable COCO metrics, then re-enable them when they are fixed.

Reply:
Is still like if we could isolate a new test under jit_compile with `@unittest.expectedFailure` of the metrics as its support status it was becoming sometime quite"mystical"

It will be easier to monitor what happens in the compiler side with an expecting failing test on the component.